### PR TITLE
allow aqt_compile to succeed if the server doesn't send back a qtrl sequence

### PR DESCRIPTION
**Pull request by richrines1**
we can use this for users without aqt configs, so the notebook will still run

(i also split up `test_read_json` into `test_read_json_only_circuits` and `test_read_json_aqt` in compiler_output_test)

### CODE REVIEW

File: cirq-superstaq/cirq_superstaq/compiler_output.py [hunk 1]
```diff
@@ -176,9 +176,18 @@ def read_json_aqt(  # pylint: disable=missing-param-doc
...
 
-    if importlib.util.find_spec(
-        "qtrl"
-    ):  # pragma: no cover, b/c qtrl is not open source so it is not in cirq-superstaq reqs
+    if "state_jp" not in json_dict:
+        warnings.warn(
+            "This output only contains compiled circuits (using a default AQT gate set). To "
+            "get back a compiled pulse sequence, you first must upload your qtrl configs using "
+            "`service.aqt_upload_configs`."
+        )
```
Comment (dowusu-antwi) [inline cirq-superstaq/cirq_superstaq/compiler_output.py:184]: nit: wording
```suggestion
        warnings.warn(
            "This output only contains compiled circuits (using a default AQT gate set). To "
            "get back the corresponding pulse sequence, you must first upload your qtrl configs "
            "using `service.aqt_upload_configs`."
        )
```

File: qiskit-superstaq/qiskit_superstaq/compiler_output.py [hunk 2]
```diff
@@ -156,9 +157,18 @@ def read_json_aqt(  # pylint: disable=missing-param-doc
...
 
-    if importlib.util.find_spec(
-        "qtrl"
-    ):  # pragma: no cover, b/c qtrl is not open source so it is not in qiskit-superstaq reqs
+    if "state_jp" not in json_dict:
+        warnings.warn(
+            "This output only contains compiled circuits (using a default AQT gate set). To "
+            "get back a compiled pulse sequence, you first must upload your qtrl configs using "
+            "`service.aqt_upload_configs`."
+        )
```
Comment (dowusu-antwi) [inline qiskit-superstaq/qiskit_superstaq/compiler_output.py:165]: nit: wording, as above
```suggestion
        warnings.warn(
            "This output only contains compiled circuits (using a default AQT gate set). To "
            "get back the corresponding pulse sequence, you must first upload your qtrl configs "
            "using `service.aqt_upload_configs`."
        )
```

Comment (richrines1) [reply qiskit-superstaq/qiskit_superstaq/compiler_output.py]: done.

**Review by bharat-thotakura — APPROVED** [review]
LGTM

**Pull request merged**